pages/ksphere/konvoy/latest/install/install-gcp/advanced-provisioning/index.md

```diff
@@ -0,0 +1,267 @@
+    use this backend unless the backend configuration changes.
+    ```
+
+    This output in this example indicates that Terraform has successfully merged content from the `backend.tf` resource file and will store the state file in Goocle Cloud Storage.
+
+## Network
+It is possible to use an existing network if so desired.
+To do so you must modify the `cluster.yaml` file and change the `ProvisionerConfig` in the following way:
+
+```yaml
+kind: ClusterProvisioner
+apiVersion: konvoy.mesosphere.io/v1beta1
+metadata:
+  name: konvoy
+spec:
+  provider: gcp
+  gcp:
+    region: us-west1
+    zones:
+    - us-west1-a
+    - us-west1-b
+    - us-west1-c
+    network:
+      selfLink: projects/my-project-id/global/networks/my-predefined-network
+...
+```
+
+### Internal only kube-apiserver LB / Forwarding rule
+
+It is **not** possible to set the kube-apiserver LB to be `internal`.
+The GCP forwarding rules do not allow a request from a backend member being directed to another backend member internally.
+This behaviour is documented [here][forwading_rules_internal].
+
+### Subnets
+An existing network may already contain `subnets` for use, you may define them in the following way:
+
+```yaml
+...
+spec:
+  provider: gcp
+  gcp:
+    region: us-west1
+    zones:
+    - us-west1-a
+    - us-west1-b
+    - us-west1-c
+    network:
+      selfLink: projects/my-project-id/global/networks/my-predefined-network
+  nodePools:
+  - name: worker
+    count: 8
+    machine:
+      rootVolumeSize: 80
+      rootVolumeType: pd-ssd
+      imagefsVolumeEnabled: true
+      imagefsVolumeSize: 160
+      imagefsVolumeType: pd-ssd
+      type: n1-standard-8
+      gcp:
+        subnetIDs:
+        - existing-subnet-for-konvoy
+  - name: control-plane
+    controlPlane: true
+    count: 3
+    machine:
+      rootVolumeSize: 80
+      rootVolumeType: pd-ssd
+      imagefsVolumeEnabled: true
+      imagefsVolumeSize: 160
+      imagefsVolumeType: pd-ssd
+      type: n1-standard-4
+      gcp:
+        subnetIDs:
+        - existing-subnet-for-control-plane-konvoy
+  - name: bastion
+    bastion: true
+    count: 1
+    machine:
+      rootVolumeSize: 80
+      rootVolumeType: pd-ssd
+      imagefsVolumeEnabled: true
+      imagefsVolumeSize: 160
+      imagefsVolumeType: pd-ssd
+      type: n1-standard-4
+      gcp:
+        subnetIDs:
+        - other-subnet
+...
+```
+
+Failure to define any subnets will mean that Konvoy will attempt to create subnets to cover missing nodepools.
+That could lead to collisions in CIDR blocks and failure to deploy; in that case we recommend a full list of subnets be known along with the nodepools desired.
+
+For the most part the nodepools created should exist in a private network configuration, which is Konvoy's default approach.
+Bastion hosts allow for secure access to your cluster, but since they do need to be accessed externally they should be deployed with a subnet where public IPs are created.
```

Comment:
```suggestion
Bastion hosts allow for secure access to your cluster. Since they need to be accessed externally, they should be deployed with a subnet where public IPs are created.
```